odoo/addons/base/models/ir_qweb_fields.py

```diff
@@ -560,7 +559,8 @@ def value_to_html(self, value, options):
                 v*secs_per_unit, threshold=1, locale=locale)
             if section:
                 sections.append(section)
-
+        if sign:
+            sections.insert(0, sign)
         return u' '.join(sections)
```

Comment:
```suggestion
        return sign + ' '.join(sections)
```
so it is similar to above